src/Matrix.cpp

```diff
@@ -296,6 +353,21 @@ int* Matrix::getIA() {
 
 
 /**
+ * @brief Get the IA component (excluding the diagonal) of the CSR form of the
```

Comment:
Perhaps it might be nice to include a `@details` comment block describing what the `IA` component physically represents?